scripts/composer/cleanup-composer

```diff
@@ -59,19 +59,20 @@ function copy_files() {
 function copy_bedrock_muplugins() {
   # Check for any of the bedrock mu-plugins.
   if [[
-      ( ! -f "web/wp-content/mu-plugins/bedrock-autoloader.php" ) ||
-      ( ! -f "web/wp-content/mu-plugins/register-theme-directory.php" )
+      ( ! -f "web/wp-content/mu-plugins/bedrock-autoloader.php" ) &&
+      ( -f "vendor/roots/bedrock/web/app/mu-plugins/bedrock-autoloader.php" )
   ]]; then
-    echo "📂 💨  Copying Bedrock mu-plugins 1/2..."
-    if [ ! -f "web/wp-content/mu-plugins/bedrock-autoloader.php" ]; then
-      cp -v vendor/roots/bedrock/web/app/mu-plugins/bedrock-autoloader.php web/wp-content/mu-plugins/
-    fi
-
-    echo "📂 💨  Copying Bedrock mu-plugins 2/2..."
-    if [ ! -f "web/wp-content/mu-plugins/register-theme-directory.php" ]; then
-      cp -v vendor/roots/bedrock/web/app/mu-plugins/register-theme-directory.php web/wp-content/mu-plugins/
-    fi
+    echo "📂 💨  Copying Bedrock mu-plugin bedrock-autoloader..."
+    cp -v vendor/roots/bedrock/web/app/mu-plugins/bedrock-autoloader.php web/wp-content/mu-plugins/
```

Comment:
Can we just use `roots/bedrock`, and overlay Pantheon components onto the site with composer/installers and scaffolding, so that we don't need this cleanup script? Or at least so that we need less of this cleanup script?